src/routes/subscriptions.rs

```diff
@@ -32,59 +32,126 @@ impl TryFrom<FormData> for NewSubscriber {
     }
 }
 
+pub enum SubscribeError {
```

Comment:
All the possible errors that can occur when hitting the subscriptions endpoint. The idea is to log errors that are more specific in nature so its easier to debug when those errors occur